src/leo-server-app/src/models/schedule.ts

```diff
@@ -0,0 +1,29 @@
+import mongoose from "mongoose";
+import { ScheduleType } from "../types/schedule";
+
+const Schema = mongoose.Schema;
+
+const scheduleSchema = new Schema(
+  {
+    commands: Schema.Types.Mixed,
+    executionTimestamp: {
+      type: Date,
+      default: Date.now,
+    },
+    satellite: {
+      type: mongoose.Schema.Types.ObjectId,
```

Comment:
can we use a string here instead? see either NORAD id or int'l code here: https://www.n2yo.com/database/?name=NEUDOSE#results

whats the benefit of using a objectid?

Reply:
The type ObjectId is how mongoose identifies each document/record, initially i had it as string but it errored out since it uses ObjectId. Is Norad id just a unique identifier for a satellite? If so, I was thinking of adding another field for the norad id

Reply:
yea i think adding another field for intl code is the move (dont need noradcode since intl is more general)

Reply:
added